index.html

```diff
@@ -87,7 +87,35 @@ <h1 class="logo-brand" itemprop="brand">MACILE</h1>
       </div>
     </div>
   </nav>
-
+  <div class="teaser">
+    <h1 class="teaser-item__tittle teaser-item__main-title">OUR NEWS</h1>
+    <div class="teaser-item">
+      <span class="teaser-item__name teaser-item__name_pink">MACILE NEWSLETTER</span>
+      <div class="teaser-item__content">
+       <span class="teaser-item__date teaser-item__date_pink">July-October, 2019</span>
+       <h1 class="teaser-item__tittle">We are the heroes</h1>
+       <p class="teaser-item__text">"We are the heroes" is the first digital newsletter of MACILE, a non-profit organization based on learning about Math, Science, Engineering and Language.
+       </p>
+       <a class="teaser-item__botton teaser-item__botton_pink">Read this issue</a>
+      </div>
+      <div class="teaser-item__img">
+       <img src="/images/newsletter/graduation_20191026MA-original.jpg" itemprop="image">
+      </div>
+    </div>
+    <div class="teaser-item teaser-item__image-first">
+      <span class="teaser-item__name teaser-item__name_blue">MACILE ACADEMY BULLETIN</span>
+      <div class="teaser-item__content">
+       <span class="teaser-item__date teaser-item__date_blue">Summer 2007-2008</span>
+       <h1 class="teaser-item__tittle">We are the Adventurers</h1>
+       <p class="teaser-item__text">The MACILE Summer Academy is a rigorous five week program in grades 6 to 12. The curriculum includes math, science, engineering, technology and language.
+      Courses and activities are designed to stimulate curiosity, creativity and passion for learning; also to build the students' skills, analytical abilities and rational thinking. The program was piloted in july 2007 and 2008.</p>
+       <a class="teaser-item__botton teaser-item__botton_blue">Read this issue</a>
+      </div>
+      <div class="teaser-item__img teaser-item__img_image-first">
```

Comment:
I like how you created a teaser-item level class `teaser-item__image-first`. When I'm creating modifier classes or classes that are used to switch the visual representation of a module, I find it is easier to conceptualize if the class is added to the highest level parent and then we use CSS chaining to affect the items we want to target.  This saves having to add individual modifier classes to each element. 

So to target the image we would use...

```css
.teaser-item_image-first .teaser-item__img { }

/* If we need to target another element */
.teaser-item_image-first .teaser-item-name {}
```

Also, we should be consistent if we use `img` or `image`. That way the developer doesn't have to guess. What are we using in other parts of the code base? We should follow that pattern.

Reply:
I can't understand what this means.